assets/js/blocks/mini-cart/utils/data.ts

```diff
@@ -13,8 +13,8 @@ export const updateTotals = ( totals: [ string, number ] | undefined ) => {
 		return;
 	}
 	const [ amount, quantity ] = totals;
-	const miniCartButtons = document.querySelectorAll(
-		'.wc-block-mini-cart__button'
+	const miniCarts = document.querySelectorAll(
+		'.wc-block-mini-cart.wp-block-woocommerce-mini-cart'
```

Comment:
I'd suggest keeping this in line with the code located in `frontend.ts`. Maybe it's also worth aborting if no mini cart blocks are found?

```suggestion
	const miniCartBlocks = document.querySelectorAll( '.wc-block-mini-cart' );
	if ( miniCartBlocks.length === 0 ) {
		return;
	}
```